### ISSUE TITLE
Mask Detector in Live Cameras[EXE]

### ISSUE THREAD

**Issue by mlunghi** [gimseng/99-ML-Learning-Projects#83]
#### Learning Goals
[Learning goals, bulleted/numbered list is preferred]
[e.g. learn the concept and the use of train/validation/test dataset using scikit-learn ]
Learn to preprocess images, use a new mobile neural network architecture, learn tensorflow.

### Exercise Statement
[Explain and describe what the exercise is]
[e.g. apply simple random-forest model to classify titanic survivability from titanic data ]
Apply Mobile Net V2 model to detect whether someone is wearing a mask from live video.

### Prerequisites 
[Prerequisites, in terms of concepts or other exercises in this repo]
[e.g. random-forest model, stochastic gradient descent, exercise #32]
Python

### Data source/summary:
[Provide a succinct summary of what the data is and where it is from] 
[e.g. This involves covid19 fatality dataset from John Hopkin's website (links..) ]
Mask images form Github.

### (Optional) Suggest/Propose Solutions
[e.g. I have the solution using PyTorch, will be happy to create pull request to include the exercise statement/solution]
[e.g. I think chapter 3 of A. Geron's textbook works out the solution for this exercise]
[e.g. fast.ai's chapter 5 has the perfect solution for this]
I have the solution in scikit learn, will be happy to share

### (Optional) Further Links/Credits to Relevant Resources:
[e.g. This exercise and solution's proposal came from a lab session from DL2020]

**Comment by gimseng** [comment]
@mlunghi This is a good idea ! Feel free to create the project. The latest available folder is `008`, so feel free to create a pull request when you are ready.

**Comment by MrPotato-00** [comment]
Hello, I want to work on this issue. Can you assign this to me?
